server/container/guice/cassandra-guice/src/main/java/org/apache/james/modules/mailbox/ElasticSearchMailboxModule.java

```diff
@@ -93,6 +98,17 @@ protected Client provideClientProvider(FileSystem fileSystem, AsyncRetryExecutor
         return client;
     }
 
+    private PropertiesConfiguration getElasticSearchConfiguration(PropertiesProvider propertiesProvider) throws ConfigurationException {
+        try {
+            return propertiesProvider.getConfiguration("elasticsearch");
```

Comment:
Then the constant "ES_CONFIG_FILE" is useless in this class? And could you take a look at ESMetricReporterModule also?